Search, One-page forms and User mgmt section review (#10)
ezsystems/commerce-documentation: docs/guide/one_page_forms/one_page_forms_api/one_page_forms_api.md

## Form validators

> Besides standard [Symfony validators,](http://symfony.com/doc/current/validation.html)

Reviewer: ```suggestion
Besides standard [Symfony validators,](http://symfony.com/doc/3.4/validation.html)
```

eZ Commerce offers additional validators.

### ZIP validator

> The post code (ZIP) validation rules are different for each country.

Reviewer: ```suggestion
The ZIP code validation rules are different for each country.
```

> To define the different code pattern for each post code a new service is defined.

Reviewer: ```suggestion
To define the different code pattern for each ZIP code a new service is defined.
```

!!! note

>     This validator validates the post code based on the submitted country.

Reviewer: ```suggestion
    This validator validates the ZIP code based on the submitted country.
```

             POST_CODE_RULE: ".*"
```

This new file must be imported in `silver.eshop.yml`:

``` yaml
- { resource: "ses_patern_zip.yml"}
```

### Email validator

A custom email validator extends the standard Symfony validation.

#### NonExistingEmailValidator

`NonExistingEmailValidator` checks if a user with the given email already exists.

#### User searching

You can set the Location where Users are contained in the following configuration:

``` yaml
ses_ez_helper.default.parent_location_id_users_members: 12
```

### Phone/fax number validator

> A phone and/or fax validator verify a valid number format. The folowing formats are supported:

Reviewer: ```suggestion
A phone and/or fax validator verifies a valid number format. The following formats are supported:
```
